app/main/index.js

```diff
@@ -3,17 +3,28 @@ const path = require('path');
 const fs = require('fs');
 const os = require('os');
 const electron = require('electron');
-const {app} = require('electron');
+const {
+	app
+} = require('electron');
 const ipc = require('electron').ipcMain;
-const {dialog} = require('electron');
+const {
+	dialog
+} = require('electron');
+const https = require('https');
+const http = require('http');
 const electronLocalshortcut = require('electron-localshortcut');
 const Configstore = require('configstore');
 const JsonDB = require('node-json-db');
 const isDev = require('electron-is-dev');
 const tray = require('./tray');
 const appMenu = require('./menu');
-const {linkIsInternal, skipImages} = require('./link-helper');
-const {appUpdater} = require('./autoupdater');
+const {
```

Comment:
Same. Keep in mind that all the exported variables should be in one line.
`const {linkIsInternal, skipImages} = require('./link-helper');`